deploy/cloudbuild-release.yaml

```diff
@@ -19,7 +19,16 @@ steps:
     args: ["build", "-f", "helm-release/Dockerfile",
            "-t", "gcr.io/kritis-project/helm-release:$TAG_NAME", "."]
   - name: "gcr.io/kritis-project/helm-release:$TAG_NAME"
-
+  # Build container to build resolve-tags
+  - name: "gcr.io/cloud-builders/docker"
+    args: ["build", "-f", "deploy/Dockerfile_resolve",
+           "-t", "resolve-tags", "."]
+  # Do the go build
+  - name: "resolve-tags"
+    args: ["make", "out/resolve-tags"]
```

Comment:
is there a reason for the cloudbuild-release.yaml making only one (the linux) resolve tags instead of releasing them all for each platform (like cloudbuild.yaml)?

Reply:
Oh no that's my mistake, should be fixed now.